packages/filesystem/src/browser/file-tree/file-tree-model.ts

```diff
@@ -121,6 +121,10 @@ export class FileTreeModel extends TreeModelImpl implements LocationService {
      * Move the given source file or directory to the given target directory.
      */
     async move(source: TreeNode, target: TreeNode) {
+        if (source.parent && source.parent.name === 'WorkspaceRoot') {
+            // do not support moving the root folder
```

Comment:
That tests for any root folder, correct? If so, the comment on the next line could be adjusted. i.e. "the root" -> "a root"